src/process.c

```diff
@@ -771,6 +771,174 @@ cc_free_pointer(gpointer str)
 	g_free(str);
 }
 
+GHashTable* mac_hash;
+
+void
+hash_mac_and_store_bdf(GHashTable* hash, gpointer mac, gpointer bdf)
+{
+       g_hash_table_insert(hash, mac, bdf);
+}
+
+static gboolean
+get_mac_and_bdf(gchar *dirname, struct cc_oci_device *dev_info, gchar **mac)
+{
+	gchar *tmp_path = NULL;
+	gchar *buffer = NULL;
+	gchar *sym_link = NULL;
+	GError *link_error = NULL;
+	gchar **tokens = NULL;
+	gchar **driver_tokens = NULL;
+	gchar **mac_from_file = NULL;
+
+	gboolean retval = false;
+	guint idx=0;
+
+	g_debug("\n=== get mac and bdf for device %s", dirname);
```

Comment:
I'd drop the `\n` as it will look very odd in the logs and confuse tools that parse the logs.